### PR TITLE
Feature/tweak download links

### PR DESCRIPTION
**Pull request by kfilip94**
This PR fixes download links, adjusting them to the updated URL structure of the data-explorer (slugs instead of ids)
[PIVOTAL](https://www.pivotaltracker.com/story/show/172835211)

**Test:** To test it check all download links that lead to the data-explorer page

### CODE REVIEW

**Comment by kfilip94** [comment]
Right, I wasn't aware of this feature there...  I will take care of it today, thanks!

**Comment by kfilip94** [comment]
Hey @Bluesmile82 , this is ready again now. Two issues that I've spotted:
1.  `Gases` filters on the Historical Emissions page don't parse very well, because on the page we use `kebabCase` function which for some cases parses the slugs differently e.g. `CH4` to `ch-4` instead of `ch4` on the backend, more about this [here](https://github.com/lodash/lodash/issues/2425)
2. Historical Emissions and years filters - `Visualize in...` doesn't pass years to the emissions page, we would need to implement that in the another PR

I'll add these two issues to Pivotal

**Comment by simaob** [comment]
Álvaro, when you can please review this again.

**Comment by Bluesmile82** [comment]
Oh! I see that this is pointing to data-explorer-url instead of develop. That is why we don't have conflicts

File: app/javascript/app/components/ndcs/lts-explore-map/lts-explore-map-selectors.js [hunk 1]
```diff
@@ -181,10 +181,26 @@ export const getPathsWithStyles = createSelector(
   }
 );
 
-export const getLinkToDataExplorer = createSelector([getSearch], search => {
-  const section = 'lts-content';
-  return generateLinkToDataExplorer(search, section);
-});
+// export const getLinkToDataExplorer = createSelector([getSearch], search => {
```
[the remark this answers is not in the document]
Comment (kfilip94) [reply app/javascript/app/components/ndcs/lts-explore-map/lts-explore-map-selectors.js]: Done ;)

**Comment by kfilip94** [comment]
Thanks for checking it, I removed commented bit as you suggested. I'll merge it now to the parent branch `feature/data-explorer-url` and then will open a PR with the rebased codebase to develop 👍

**Pull request merged**